### PR TITLE
gemfile, package json changes, everything is working

### PR DESCRIPTION
**Pull request by Lilmcg**

### CODE REVIEW

**Review by WalkerTexas-coder — COMMENTED** [review]
Are your landing page user stories complete?

**Comment by Lilmcg** [comment]
> Are your landing page user stories complete?

Yes. Need to start coding it now

**Comment by Lilmcg** [comment]
> > Are your landing page user stories complete?
> 
>  No. We just spoke and this PR is not necessary after all. I will continue working on this branch for landing page code

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
-    console.log("sign out:", this.props.sign_out_route)
     return (
-      <>
-        <h1>Kick Ass Apartment App!</h1>
-      </>
+      <Router>
+        <Header {...this.props} />
+        <Routes>
+          <Route exact path="/" element={<Home />} />
+          <Route path="/users/sign_in" />
```
Comment (sjproctor) [inline app/javascript/components/App.js:20]: The user path should come from your devise props?

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -1,17 +1,26 @@
-import React from "react"
+import React, { Component} from 'react'
+import Home from './pages/Home'
+import Header from './components/Header'
+import Footer from './components/Footer'
 import PropTypes from "prop-types"
```
Comment (sjproctor) [inline app/javascript/components/App.js:5]: This proptypes line can be removed

File: app/javascript/components/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
+import { NavLink } from 'react-router-dom'
+
+export default class Footer extends Component {
+    render() {
+        return (
+            <footer>
+        <ul>
```
Comment (sjproctor) [inline app/javascript/components/components/Footer.js:8]: Keep an eye on your indentation!

**Review by sjproctor — APPROVED** [review]
Great job, just a few notes to keep in mind.

**Pull request merged**